src/components/shared/filterDropdownButton/dateFieldGroup.test.js

```diff
@@ -75,21 +75,21 @@ describe('DateFieldGroup', () => {
       wrapper.find('.dateToInput input').simulate('change', { target: { name: 'dateTo', value: '12.12.16' } });
       jest.advanceTimersByTime(300);
       wrapper.update();
-      expect(wrapper).toContainMatchingElement('.feedback.show');
+      expect(wrapper).toContainMatchingElement('.feedback');
```

Comment:
💅 This is not testing anything anymore. I fixed the same problem in `amountFieldGroup.js` in https://github.com/LiskHQ/lisk-hub/pull/2601/commits/d9bc6ad522dbcc494febc3f1a41d71d246459bc6